### DISCUSSION TITLE
May I join the team?

### DISCUSSION THREAD

**Discussion by shekhawat2** [ROM-builders/temporary#18807] [General]
Github: https://github.com/Shekhawat2 ( @shekhawat2 )
Telegram: https://t.me/shekhawat2

PR:
https://github.com/ROM-builders/temporary/pull/18372

Successful Builds:
https://cirrus-ci.com/github/ROM-builders/temporary/pull%2F18372

**Comment by Apon77** [comment]
seems like your account is inactive

**Reply by shekhawat2** [reply]
which account sir? i am active on both github and tg

**Reply by shekhawat2** [reply]
> seems like your account is inactive

Please sir. add me. 🥹

**Reply by Apon77** [reply]
> which account sir? i am active on both github and tg

I can't see any update in github since 2 months!

**Reply by shekhawat2** [reply]
Ok bro. your wish.
